### DISCUSSION TITLE
Are any of the gpt4all models supported?

### DISCUSSION THREAD

**Discussion by ralyodio** [ggml-org/llama.cpp#1514] [Q&A]
https://gpt4all.io/models/models.json

**Comment by BetaDoggo** [comment]
Only the ones labeled as llama based. Additionally I believe that the models currently working in gpt4all won't work in the current llama.cpp because of the latest breaking change in the model format. The Bloke(Tom Jobbins) has a large number of supported models on his Huggingface, including gpt4all. Be aware that these may also break soon if #1508 is merged without support for previous models.
